ui/app/src/utils/path.ts

```diff
@@ -41,6 +41,27 @@ export function getPathFromPreviewURL(previewURL: string) {
   return `/site/website${pagePath}`;
 }
 
+export function getPreviewURLFromPath(baseUrl: string, path: string) {
+  let url = '';
+  if(path.endsWith('.xml')) {
+    url.replace('.xml', '.html');
+  }
+  url.replace('/site/website','');
+  return baseUrl+url;
+}
+
+export function isEditableFormAsset (path: string) {
```

Comment:
1. Move this to a new util called `content.ts`
2. Rename function to `isEditableAsset` (why form?)
3. See https://github.com/craftercms/studio-ui/pull/1488/files. Add/test. (Will need to refactor this to a common place later on.)